core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/util/DefaultIItemResultSet.java

```diff
@@ -66,12 +69,17 @@
 		aggregators.addAll(query.getAggregators());
 		info = new HashMap<>(attributes.size() + aggregators.size());
 		initializeMetadata();
-		calculateData(items);
+		try {
+			calculateData(items);
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
```

Comment:
Don't swallow the interrupted exception, if this happens, this mean the code calling awaitTermination has been interrupted, the code should make sure the current thread is interrupted.